src/Lykke.Service.LiteCoin.Sign/Lykke.Service.LiteCoin.Sign.csproj

```diff
@@ -15,9 +15,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Lykke.Common" Version="7.0.1" />
-    <PackageReference Include="Lykke.Common.ApiLibrary" Version="1.9.0" />
-    <PackageReference Include="Lykke.Logs" Version="5.1.0" />
+    <PackageReference Include="Lykke.Common" Version="7.0.3" />
+    <PackageReference Include="Lykke.Common.ApiLibrary" Version="1.10.1" />
+    <PackageReference Include="Lykke.Logs" Version="5.2.0" />
     <PackageReference Include="Lykke.Service.Session" Version="1.0.33" />
     <PackageReference Include="Lykke.SettingsReader" Version="2.4.1" />
     <PackageReference Include="Lykke.SlackNotification.AzureQueue" Version="2.0.5" />
```

Comment:
Not sure if we still need this reference here.

Reply:
services.AddEmptyLykkeLogging() requires Lykke.Logs